packages/beluga/beluga.1.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "Implementation of contextual modal logic for reasoning with higher-order abstract syntax"
+maintainer: "jacob.errington@mail.mcgill.ca"
+authors: [
+  "Brigitte Pientka"
+  "Joshua Dunfield"
+  "Andrew Cave"
+  "Jacob Thomas Errington"
+  "Junyoung Clare Jang"
+]
+license: "GPLv3"
+homepage: "https://github.com/Beluga-lang/Beluga"
+bug-reports: "https://github.com/Beluga-lang/Beluga/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "1.1"}
+  "gen"
+  "sedlex"
+  "extlib"
+  "dune-build-info"
+  "linenoise"
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "exec" "./TEST"] {with-test}
+  ["dune" "exec" "./TEST" "--" "--" "+htmltest"] {with-test}
+  ["dune" "exec" "./TEST" "--" "--" "+sexp"] {with-test}
+]
+run-test: [
+  ["./TEST"]
+  ["./TEST" "--" "+htmltest"]
+]
```

Comment:
The `./TEST` calls seem to repeat themselves twice (in build and in run-test). Is that expected?